learn/guides/ballerina-shell.md

```diff
@@ -189,12 +250,48 @@ The `/reset` command can be used to clear the current definitions in the memory
 
 ```bash
 =$ /dclns                                                                                                                                                                                                          
-| ('Pet) type Pet record {    string name;};
-| ('PI) const PI = 3.14;
+| (subtract) function subtract(int a, int b) returns...                        return a - b ;}
+| (add) function add(int a, int b) returns int ...                         return a + b;}
+| (mulitply) function mulitply(int a, int b) returns...                         return a * b;}
 
 =$ /reset                                                                                                                                                                                                            
 | REPL state was reset.
 
 =$ /dclns                                                                                                                                                                                                          
 | 
 ```
+
+## Stopping the Shell
+
+Execute the `/exit` command to exit the Shell.
+
+```bash
+=$ /exit
+| Bye!!!                                          
+```
+
+## Help
+
+### View available topics
+
+Execute the `/help topics` command to view available topics.
+
+### View specific topic details
+
+Execute the `/help <TOPIC>` command to view details related to mentioned topic.
+
+Consider following example
```

Comment:
```suggestion
For example, see below.
```

Reply:
Updated in 2a10777065a5455f4155b3fbdd75b7a70f169c75